src/models/Settings.js

```diff
@@ -25,6 +32,10 @@ function (Okta, Errors, BrowserFeatures) {
       ConfigError = Errors.ConfigError,
       UnsupportedBrowserError = Errors.UnsupportedBrowserError;
 
+  var assetBaseUrlTpl = Okta.tpl(
+    'https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/{{version}}'
```

Comment:
Our customers using i18n features in Preview will need to know this, but there's nothing wrong with the solution. Maybe deploying to the cdn on every release, including trex, is a strategy we should consider (at least for sdks).